exploration/open-close-expressions.md

```diff
@@ -220,3 +220,68 @@ Rendered as React this could become:
 _What other solutions are available?_
 _How do they compare against the requirements?_
 _What other properties they have?_
+
+### Do Nothing
+
+We could not provide any special support for spans and markup elements,
+and instead delegate the problem completely to tools and downstream processing layers.
+
+```
+This is <html:strong>bold</html:strong> and this is <html:img alt="an image">.
+```
+
+#### Pros:
+
+* No work required from us right now.
+* It's HTML.
+* The least surprising syntax for developers and translators.
+* Some CAT tools already support HTML in translations.
```

Comment:
```suggestion
* Some CAT tools already support HTML and other markup in translations.
* Supports other markup or templating syntaxes with no additional effort (it's just strings)
```

Reply:
I like this suggestion, but one comment: `supports` is saying too much. This approach *doesn't impede* markup expressed as regular text, but also does nothing to *support* it. It doesn't exist in the AST, in the data model, nor doesn't affect the result of `formatToString` not `formatToParts`.

Reply:
I'm going to commit this suggestion and propose a slightly different wording in a follow-up comment.